### PR TITLE
Add navstar, and its deps; Bump minuteman

### PR DESCRIPTION
**Pull request by sargun**

### CODE REVIEW

**Comment by sargun** [comment]
Relies on #197

**Comment by spahl** [comment]
:ship: This is the base for virtual networks.

File: packages/erlang/build [hunk 1]
```diff
@@ -18,7 +18,6 @@ cat <<EOF > $service
 Description=Erlang Port Mapping Daemon: DC/OS Erlang Port Mapping Daemon
 
 [Service]
-User=dcos_epmd
```
Comment (cmaloney) [inline packages/erlang/build]: Why is this dropping the user? It should remain non-root...

File: packages/navstar/build [hunk 2]
```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+set -x
+export LD_LIBRARY_PATH=/opt/mesosphere/lib:/opt/mesosphere/active/libsodium/lib
+export CFLAGS="-I/opt/mesosphere/include -I/opt/mesosphere/active/libsodium/include"
```
Comment (cmaloney) [inline packages/navstar/build]: Everything in "/lib" inside libsodium should end up at /opt/mesosphere/lib, so this and he lib line shouldn't be neededm

File: packages/navstar/build [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+export LDFLAGS="-L/opt/mesosphere/lib -L/opt/mesosphere/active/libsodium/lib -Wl,-rpath=/opt/mesosphere/active/libsodium/lib"
+export CXXFLAGS=CFLAGS
+
+source /opt/mesosphere/environment
```
Comment (cmaloney) [inline packages/navstar/build]: Is start with this. It will set LD_LIBRARY__PATH as needed

File: packages/navstar/build [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+mkdir -p $(dirname $service)
+
+cat <<EOF > $service
+[Unit]
+Description=Navstar: A distributed systems & network overlay orchestration engine
+After=dcos-gen-resolvconf.service
+After=dcos-epmd.service
+
+[Service]
+Restart=always
```
Comment (cmaloney) [inline packages/navstar/build]: Can we have this not run as root?

File: packages/erlang/build (hunk 1 above)
Comment (sargun) [inline packages/erlang/build]: I can fix this. No idea how it got in there

File: packages/navstar/build (hunk 4 above)
Comment (sargun) [inline packages/navstar/build]: It needs CAP_NET_ADMIN

Comment (sargun) [inline packages/navstar/build]: I dropped the caps.

**Comment by sargun** [comment]
@cmaloney @spahl -- any other issues?
We plan to add integration tests once Marathon has the ability to launch apps on containers. 

Proof of tests:
https://circleci.com/gh/dcos/navstar/12

Documentation will come with EA.

**Comment by spahl** [comment]
Looks good to me.

File: packages/navstar/build [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+RestartSec=5
+WorkingDirectory=${PKG_PATH}/navstar
+EnvironmentFile=/opt/mesosphere/environment
+ExecStartPre=/bin/ping -c1 ready.spartan
+ExecStartPre=/usr/bin/env mkdir -p /var/lib/dcos/navstar/mnesia
+ExecStartPre=/usr/bin/env mkdir -p /var/lib/dcos/navstar/lashup
+ExecStart=${PKG_PATH}/navstar/bin/navstar-env foreground
+Environment=HOME=/opt/mesosphere
+User=dcos_navstar
+AmbientCapabilities=CAP_NET_ADMIN
```
Comment (spahl) [inline packages/navstar/build]: We can't use AmbientCapabilities because it is not available in older versions of systemd...

**Pull request merged**